bot/common/threads/onboarding.py

```diff
@@ -677,20 +664,16 @@ def _govrn_oboard_steps(self):
 
     def _data_retrival_steps(self):
         return (
-            Step(current=AddUserTwitterStep(self.cache))
```

Comment:
instead of prompting first for a user profile, we just ask for a link to a tweet with a specified nonce. keeping the explicit prompt for the wallet step, since if the user signs a message with an e.g. extra space, the recovered (and presumably verified) account would be completely different than the account a user wanted to associate. keeping this explicit check (for now) to avoid that case.